great_expectations/experimental/datasources/type_lookup.py

```diff
@@ -51,6 +51,13 @@ def __init__(
         __dict = __dict or {}
         super().__init__(__dict, **kwargs)
 
+    def str_values(self) -> Generator[str, None, None]:
```

Comment:
Maybe change this name to `type_names`? While these are the `dict` `values` because of the nature of this data structure, you iterate through the keys. Changing the name so it doesn't sound like it refers to something in the implementation might make it clearer for the reader.

Reply:
Much better. Done.